tests/AssertTest.php

```diff
@@ -289,8 +290,18 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
+            array('throws', array(function() { throw new Exception('test'); }), true),
         );
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
```

Comment:
effect is same, but using constant is faster than calling function and play with a strings. just a good practice @Nyholm